scylla/src/transport/cluster.rs

```diff
@@ -52,7 +52,7 @@ pub struct Datacenter {
 
 #[derive(Clone)]
 pub struct ClusterData {
-    pub(crate) initially_known_peers: Arc<Vec<SocketAddr>>, // Invariant: nonempty after Cluster::new()
+    pub(crate) initially_known_peers: Vec<SocketAddr>, // Invariant: nonempty after Cluster::new()
```

Comment:
it looks like this commit fixes an issue introduced in another commit in the same series (c12026a3acff585f20a2c8472bd9ecf562b4d6ec) - if that's indeed the case, please squash them together